app/views/layouts/printer_friendly.html.haml

```diff
@@ -1,11 +1,13 @@
 !!!
 %html{dir:language_direction}
-  %head
+  %head      
```

Comment:
please remove the blank spaces at the end of line. It my be confusing to see lines in the diff where nothing really changed.